ci/prebuilt/BUILD

```diff
@@ -100,7 +110,9 @@ cc_library(
         "thirdparty/spdlog/include/**/*.cc",
         "thirdparty/spdlog/include/**/*.h",
     ]),
+    defines = ["SPDLOG_FMT_EXTERNAL"],
```

Comment:
@kmyerson you should defer this change until the next PR, as it has a dependency on the base Docker image having fmtlib included in the prebuilts.